### ISSUE TITLE
Join error since new update

### ISSUE THREAD

**Issue by linsaftw** [WaterfallMC/Waterfall#746]
```[19:53:37 WARN] [io.netty.util.concurrent.AbstractEventExecutor]: A task raised an exception. Task: net.md_5.bungee.connection.InitialHandler$6$1@6c6e6550
java.lang.NullPointerException: Cannot invoke "net.md_5.bungee.connection.LoginResult.getProperties()" because the return value of "net.md_5.bungee.connection.InitialHandler.access$900(net.md_5.bungee.connection.InitialHandler)" is null
        at net.md_5.bungee.connection.InitialHandler$6$1.run(InitialHandler.java:573) ~[Waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:34b6d30:unknown]
        at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174) ~[Waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:34b6d30:unknown]
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167) ~[Waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:34b6d30:unknown]
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470) ~[Waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:34b6d30:unknown]
        at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:391) ~[Waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:34b6d30:unknown]
        at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:995) ~[Waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:34b6d30:unknown]
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[Waterfall.jar:git:Waterfall-Bootstrap:1.19-R0.1-SNAPSHOT:34b6d30:unknown]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]```

Local offline-mode server presenting issues on player login.

**Comment by linsaftw** [comment]
Seems updating bungeecord should fix the issue.

**Comment by eleksploded** [comment]
this was futher fixed in bungee [8c0e4b1](https://github.com/SpigotMC/BungeeCord/commit/8c0e4b1d338bab14cf972959dc020f9c3f469551)
